rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2209,6 +2209,32 @@ void getProposedFederation_whenStorageProviderReturnsProposedFederation_shouldRe
         assertEquals(proposedFederation, actualProposedFederation.get());
     }
 
+    @Test
+    void getProposedFederationCreationTime_whenStorageProviderReturnsEmpty_shouldReturnEmpty() {
+        // Act
+        Optional<Instant> actualCreationTime = federationSupport.getProposedFederationCreationTime();
+
+        // Assert
+        assertFalse(actualCreationTime.isPresent());
+    }
+
+    @Test
+    void getProposedFederationCreationTime_whenStorageProviderReturnsProposedFederation_shouldReturnCreationTime() {
+        // Arrange
+        Instant creationTime = Instant.EPOCH;
+        Federation proposedFederation = P2shErpFederationBuilder.builder()
+            .withCreationTime(creationTime)
+            .build();
```

Comment:
i think theres no need to do that, we already have a [creationTime](https://github.com/rsksmart/rskj/blob/feat/get_proposed_federation_creation_time_fed_support/rskj-core/src/test/java/co/rsk/peg/federation/P2shErpFederationBuilder.java#L41) defined in the builder :)

Reply:
Sounds good, will update

Reply:
Well, in this case it might be needed if we want to make it explicit and assert against the value intentionally set

Reply:
yes